R/PipeOpFFS.R

```diff
@@ -86,27 +86,45 @@ PipeOpFFS = R6Class("PipeOpFFS",
       dt = task$data(cols = cols)
       pars = self$param_set$get_values()
       drop = pars$drop
-      feature = pars$feature
+      features = pars$features
       left = pars$left
       right = pars$right
       assert_true(left <= right)
+      assert_list(features, types = c("character", "function"), any.missing = FALSE, unique = TRUE)
```

Comment:
somewhere we need to check that the functions have names and that all names (either the predefined strings) or the function names are unique